applications/advanced-concepts/arbitrage.md

```diff
@@ -0,0 +1,20 @@
+---
+description: Arbitrage for machine translation
```

Comment:
"arbitrage" or "engine picking" is how I think of it, but I'm not sure what the most common name is for this approach.

https://en.wikipedia.org/wiki/Hybrid_machine_translation

"hybrid" to me implies a mix of human and machine, whereas this is a mix of multiple machines.

I might call it "multi-engine machine translation".

Reply:
I think "hybrid" is the best way to describe the half-machine, half-human workflow.
"Artitrage" is probably a difficult term that misses the scope of the site. "Engine picking" seems to be the most transparent term to describe this topic.
To me, "multi-engine machine translation" takes the focus out of the process of evaluating and choosing the best fit, it just concentrates of the results.

Reply:
It's the most transparent, but I'm leery of inventing a term.

I did some searching now, as far as I can tell, "multi-engine machine translation" has the most currency in the literature.